cmd/derper/derper.go

```diff
@@ -190,29 +196,33 @@ func main() {
 	s.SetTCPWriteTimeout(*tcpWriteTimeout)
 
 	if *dev {
-		k := setec.StaticSecret(os.Getenv("TAILSCALE_DERPER_MESH_KEY"))
-		if k.GetString() != "" {
-			key, err := checkMeshKey(k.GetString())
-			if err != nil {
-				log.Fatalf("setec static failed to get key: %v", err)
-			}
-			s.SetMeshKey(key)
-		} else {
+		key, ok := os.LookupEnv("TAILSCALE_DERPER_MESH_KEY")
```

Comment:
I realized belatedly that my example was silly, if we don't care about the difference between "empty" and "unset" we could also simplify this to 

```go
key := os.Getenv("TAILSCALE_DERPER_MESH_KEY")
if key == "" {
   log.Println("No mesh key specified for dev")
} else if meshKey, err := …
```

and so on.